Library/Homebrew/cmd/config.rb

```diff
@@ -119,9 +119,15 @@ def describe_system_ruby
     s
   end
 
-  def java_version
-    java = `java -version 2>&1`.lines.first.chomp
-    java =~ /java version "(.+?)"/ ? $1 : java
+  def describe_java
+    if which("java").nil?
```

Comment:
Total nitpick but reads a little weird to me to check for `java` here but only use it in `else`. Maybe do `if .. java_home`, `elsif which(java)`, `else N/A` (if that makes sense)